stacks/spica/src/bucket/pages/index/index.component.html

```diff
@@ -115,21 +102,27 @@ <h5>
               </mat-checkbox>
             </button>
             <mat-divider></mat-divider>
-            <button
-              mat-menu-item
-              disableRipple
-              *ngFor="let property of properties"
-              [disabled]="property.name == '$$spicainternal_select' && readOnly"
-              (click)="$event.stopPropagation()"
-            >
-              <mat-checkbox
+            <ng-container *ngFor="let property of properties">
+              <button
+                mat-menu-item
+                disableRipple
+                *ngIf="
+                  property.name != '$$spicainternal_select' &&
+                  property.name != '$$spicainternal_id' &&
+                  property.name != '$$spicainternal_actions'
```

Comment:
Not required but it would be better if we define an array for these values like **alwaysDisplayedProperties**